snap/snapcraft.yaml

```
@@ -1,6 +1,8 @@
 name: charmed-openstack-exporter
 base: core22
 summary: OpenStack Exporter for Prometheus
+license: MIT
+contact: Solutions Engineering <solutions-engineering.lists.canonical.com>
```

```suggestion
contact: Solutions Engineering <solutions-engineering@lists.canonical.com>
```

I think it should be an email address?